liquidhaskell-boot/src/Language/Haskell/Liquid/UX/CmdLine.hs

```diff
@@ -524,12 +524,12 @@ withSmtSolver cfg =
                    case found of
                      Just _ -> return cfg
                      Nothing -> panic Nothing (missingSmtError smt)
-    Nothing  -> do smts <- mapM findSmtSolver [FC.Z3, FC.Cvc4, FC.Mathsat]
+    Nothing  -> do smts <- mapM findSmtSolver [FC.Z3, FC.Cvc5, FC.Cvc4, FC.Mathsat]
                    case catMaybes smts of
                      (s:_) -> return (cfg {smtsolver = Just s})
                      _     -> panic Nothing noSmtError
   where
-    noSmtError = "LiquidHaskell requires an SMT Solver, i.e. z3, cvc4, or mathsat to be installed."
+    noSmtError = "LiquidHaskell requires an SMT Solver, i.e. z3, cvc5, cvc4, or mathsat to be installed."
```

Comment:
To avoid the burden of updating this line in the future, you might want to generate it. E.g. if you derived `Bounded` and `Ord` for `SMTSolver`, you could have said `show [minBound..maxBound]` instead.